Example/Controllers/ViewController.swift

```diff
@@ -54,16 +54,16 @@ class ViewController: UIViewController {
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if segue.identifier == "DisplayMiniApp" {
-            guard let miniAppInfo = self.currentMiniAppInfo, let miniAppDisplay = self.currentMiniAppView else {
```

Comment:
In normal case that's true. Since url loaded miniapp doesn't produce miniAppInfo content, the guard would block the display. I guess I can leave that check for miniapps from the list and add an exception for url loaded ones

Reply:
The MiniApp View can work with miniAppInfo as nil and that is the case during url loading. Adding some special flag to enable miniAppInfo doesn't make sense for me. Plus there is no error handling implemented (it just doesn't display the view)